internal/strict/controls/controls.go

```diff
@@ -43,6 +43,9 @@ const (
 
 	// RequireExplicitBootstrap is the control that prevents the backend for remote state from being bootstrapped unless the `--backend-bootstrap` flag is specified.
 	RequireExplicitBootstrap = "require-explicit-bootstrap"
+
+	// CLIRedesign is the control that prevents the use of deprecated commands.
```

Comment:
```suggestion
	// CLIRedesign is the control that prevents the use of commands deprecated as part of the CLI Redesign.
```

NIT: Please address this in a different PR.